source/_integrations/slack.markdown

```diff
@@ -18,39 +18,69 @@ ha_integration_type: service
+
 4. Scroll up to `OAuth Tokens & Redirect URLs` and click `Install to Workspace`.
-5. Copy your `OAuth Access Token` and put that key into the config flow.
 
-<div class='note'>
+In `Features/OAuth and Permissions/OAuth Tokens for Your Workspace`:
 
-There is an app credential Verification Token on the Basic Settings of your app. This is **not** the API key you want.
+5. Copy the User (or Bot User) OAuth Token. Use this as 'API Key' when setting up in Home Assistant
 
-</div>
 
-### Bot posting as its own user
+## Usage
 
-It is also possible to use Slack bots as users. Just create a new bot at https://[YOUR_TEAM].slack.com/apps/build/custom-integration and use the provided token for that. You can add an icon from the frontend for Home Assistant and give the bot a meaningful name.
+### Sending Messages
 
-Don't forget to invite the bot to the room where you want to get the notifications.
+One of the easiest ways to send a message, is to create a script. You can paste in YAML and make changes in the GUI.
 
-### Icons
+You can call this script as a service. 
 
-Slack uses the standard emoji sets used [here](https://www.webpagefx.com/tools/emoji-cheat-sheet/). Alternatively a publicly accessible URL may be used.
+1. Go to Home Assistant Settings -> Automations and Scenes -> Scripts -> Add Script
+2. Click the three dots in the top right, and pick 'Edit in YAML'. Paste in the contents below.
+3. Change `YOUR_SLACK_TEAM` to the team name `(*.slack.com)`
+
+```yaml
+alias: "Notify: Slack Notification Template"
+sequence:
+  - service: notify.YOUR_SLACK_TEAM
+    data:
+      message: Fallback Text
+      target: "#test-channel"
+      title: Reminder
+      data:
+        blocks:
+          - type: section
+            text:
+              type: mrkdwn
+              text: >-
+                This is a mrkdwn section block *this is bold*, and ~this is
+                crossed out~, and <https://google.com|this is a link>
+mode: single
+```
 
-<div class='note'>
+Update the blocks array with valid Slack blocks. The easiest way to create this is using [Slack Block Kit Builder](https://app.slack.com/block-kit-builder)
 
-In order to modify your Slack bot's username and icon, you must ensure your Slack app has the `chat:write.customize` OAuth scope. See [the Slack API documentation](https://api.slack.com/methods/chat.postMessage#authorship) for more information.
+Create a duplicate of this script to use for different messages, and different channels (the door was opened in #security, the light was left on on #lights, etc).
 
-The added `notify` service will be named after the chat server the app is installed on. For example, a server named "Slack Chat" wil display as `notify.slack_chat`.
+### Icons
 
-</div>
+Slack uses the standard emoji sets used [here](https://www.webpagefx.com/tools/emoji-cheat-sheet/). Alternatively a publicly accessible URL may be used.
```

Comment:
It would be better to reference the [Slack documentation on this](https://slack.com/intl/en-gb/help/articles/202931348-Use-emoji-and-reactions#add-emoji-to-your-messages) so if they change their Emoji support, I'd hope they would update their documentation rather to a third party site direct.